.github/workflows/base-component-descriptor.yaml

```diff
@@ -138,6 +138,12 @@ jobs:
             ref: ${{ github.ref }}
           EOF
 
+          echo "adding bdba component-reference"
+          python3 -m ocm add component-ref \
```

Comment:
as we discussed this last week (out-of-bands): This is _not_ a good place to add repository-specific customations.

-> base-component-descriptor-workflow is intended to be re-usable. We _might_ add an input to pass-in component-references (which would make it more convenient for us to do pass-in via stdin, btw), or create a `.ci/component-descriptor`-callback (which base-component-descriptor will need to honour, of course).